runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process/actions/excel.ts

```diff
@@ -41,6 +41,10 @@ const excelActionsTranslations: typeof englishExcelActionsTranslations = {
     'Process.Details.Modeler.Actions.Excel.InsertColumns.ColumnsAmount': 'Liczba kolumn',
     'Process.Details.Modeler.Actions.Excel.InsertColumns.Column.Info': 'Oznaczenie literowe lub numer kolumny, np "AZ" lub 25.',
     'Process.Details.Modeler.Actions.Excel.InsertColumns.Amount.Info': 'Liczba kolumn do wstawienia.',
+    'Process.Details.Modeler.Actions.Excel.InsertRowsAfter.Label': 'Wstaw wiersze po',
+    'Process.Details.Modeler.Actions.Excel.InsertRows.RowsNumber': 'Liczba rzędów do wstawienia',
+    'Process.Details.Modeler.Actions.Excel.InsertRows.Row.Info': 'Początkowy numer rzędu, np. 5',
```

Comment:
numer początkowego wiersza will maybe sound better? rzędy -> wiersze - what do you think?